util/index.ts

```diff
@@ -525,6 +530,44 @@ export const removeLightningPrefix = (invoice: string) => {
   return invoice;
 };
 
+const generateRandomImage = async (nonce: string) => {
+  let randomImage = '';
+  try {
+    const files = await fs.readdir('images');
+    const imageFiles = files.filter(file =>
+      ['.png'].includes(path.extname(file).toLowerCase())
+    );
+
+    const randomFile = imageFiles[Math.floor(Math.random() * imageFiles.length)];
+    const fallbackImage = await fs.readFile(`images/${randomFile}`);
+
+    randomImage = Buffer.from(fallbackImage, 'binary').toString('base64');
+  } catch (fallbackError) {
+    logger.error(fallbackError);
+  }
+
+  return randomImage;
+}
+
+const generateQRWithImage = async (request, randomImage) => {
+  const canvas = createCanvas(400, 400);
+  await QRCode.toCanvas(canvas, request, {
+    margin: 2,
+    width: 400,
+  });
+
+  const ctx = canvas.getContext('2d');
+  const centerImage = new Image();
+  centerImage.src = `data:image/png;base64,${randomImage}`;
+
+  const imageSize = canvas.width * 0.2;
```

Comment:
I think a more appropriate size might be 0.3

Reply:
Set it to 0.3. Looks much better.